src/Goerli-DssSpell.t.sol

```diff
@@ -2323,131 +2100,196 @@ contract DssSpellTest is DSTest, DSMath {
         jug.drip(_ilk);
         assertEq(clip.kicks(), 0);
         if (_checkLiquidations) {
-            dog.bark(_ilk, address(this), address(this));
+            cat.bite(_ilk, address(this));
```

Comment:
I think this should be `dog.bark`, @gbalabasquer?
Is not catched in the test cause the liquidations are off with guni lp.

Reply:
I think so too

Reply:
done